plugins/YouTubeImport/YouTubeImportPlugin.php

```diff
@@ -39,8 +40,12 @@ class YouTubeImportPlugin extends Omeka_Plugin_AbstractPlugin
     /**
      * @var array Options for the plugin.
      */
-    protected $_options = array('youtube_width'=>640,'youtube_height'=>360);
+    protected $_options = array('youtube_width'=>640,'youtube_height'=>360, 'youtube_apikey'=>'AIzaSyDI8ApsA7MBIK4M1Ubs9k4-Rk7_KOeYJ5w');
```

Comment:
Should the api key be in plain sight like this?

Reply:
Not really our issue, as that key has been out in the open [here](https://github.com/UCSCLibrary/YouTubeImport/blob/c708f969d063dd343e23080b84566435271b9ccb/helpers/import.php#L20) since 2017? Because it's exposed, it's not even useful for testing as a default though. Maybe setting it to empty is cleaner.